TreeList: add description for rowRenderingMode (T1170183) (#5167)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxTreeList/1 Configuration/scrolling/rowRenderingMode.md

> Unlike the [mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/scrolling/#mode) property, which defines when to load data for rows, this property defines when to render them. The following rendering modes are available: 

Reviewer: ```suggestion
Unlike the [mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/scrolling/#mode) property that defines when to load data for rows, this property defines when to render rows. The following rendering modes are available: 
```

- *"standard"*        

> Renders all the loaded rows at once.

Reviewer: ```suggestion
Renders all the loaded rows simultaneously.
```

- *"virtual"*     

> Renders only those rows that get into the viewport. Use this mode if the [pageSize](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/paging/#pageSize) is large. Specify the component's [height](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#height) and height of its container if you use the *virtual* mode. 

Reviewer: ```suggestion
Renders only those rows that come into the viewport. Use this mode if the [pageSize](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/paging/#pageSize) is large. Specify the component's [height](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#height) and the height of its container if you use the *virtual* mode. 
```
